pywbem/_tupleparse.py

```diff
@@ -150,13 +150,14 @@ def pcdata(self, tup_tree):
         """
         try:
             data = u''.join(tup_tree[2])
-
         except TypeError:
-            raise CIMXMLParseError(
+            new_exc = CIMXMLParseError(
```

Comment:
This one confuses me in that I thought that __cause__ was only involves when the
raise was of the form

raise new_exc from original_exc

Reply:
It is the portable and more recent alternative to that.